docs/tutorials/python/compliance/verify-credential.md

```diff
@@ -0,0 +1,214 @@
+- You should have a basic understanding of the XRP Ledger
+- The credential you want to verify should exist in the ledger already, and you should know the addresses of both the issuer and the holder, as well as the official credential type you want to check.
+<!-- Uncomment when both are merged:
+     - For sample code showing how to create credentials, see [Build a Credential Issuing Service](../build-apps/credential-issuing-service.md).
+-->
+
+## Setup
+
+First, download the complete sample code for this tutorial from GitHub:
+
+- {% repo-link path="_code-samples/verify-credential/py/" %}Verify Credential sample code{% /repo-link %}
+
+Then, in the appropriate directory, set up a virtual environment and install dependencies:
+
+```sh
+python -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+```
+
+This should install an appropriate version of the `xrpl-py` library. There are no other dependencies for this tutorial outside of the Python standard library.
+
+## Overview
+
+The Verify Credential sample code consists of one file, `verify_credential.py`, containing two main parts:
+
+1. A function, `verify_credential(...)` which can be called with appropriate arguments to verify that a credential exists and is valid. This function can be imported into other code to be used as part of a larger application.
+2. A commandline utility that can be used to verify any credential.
+
+## Usage
+
+To test that you have the code installed and working properly, you can run the commandline utility with no arguments to check the existence of a default credential on Devnet, such as:
+
+```sh
+python verify_credential.py
+```
+
+If all goes well, you should see output such as the following:
+
+```text
+Encoded credential_type as hex: 6D795F63726564656E7469616C
+Looking up credential...
+{'ledger_index': 'validated', 'method': 'ledger_entry', 'api_version': 2, 'credential': {'subject': 'rsYhHbanGpnYe3M6bsaMeJT5jnLTfDEzoA', 'issuer': 'rEzikzbnH6FQJ2cCr4Bqmf6c3jyWLzkonS', 'credential_type': '6D795F63726564656E7469616C'}, 'binary': False}
+Found credential:
+{'CredentialType': '6D795F63726564656E7469616C', 'Flags': 65536, 'Issuer': 'rEzikzbnH6FQJ2cCr4Bqmf6c3jyWLzkonS', 'IssuerNode': '0', 'LedgerEntryType': 'Credential', 'PreviousTxnID': '7D1257779E2D298C07C7E0C73CD446534B143FBD1F13DB268A878E40FD153B9A', 'PreviousTxnLgrSeq': 803275, 'Subject': 'rsYhHbanGpnYe3M6bsaMeJT5jnLTfDEzoA', 'SubjectNode': '0', 'index': '9603F0E204A8B1C61823625682EB0ECE98A4ECF22FF46CD4845FA9BFA3606B24'}
+Credential is valid.
+```
+
+If the code instead reports that the credential was not found when called with no arguments, it's possible that the example credential has been deleted or the Devnet has been reset. If you have another credential you can verify, you can provide the details as commandline arguments. For example:
+
+```sh
+python verify_credential.py rsYhHbanGpnYe3M6bsaMeJT5jnLTfDEzoA rsYhHbanGpnYe3M6bsaMeJT5jnLTfDEzoA my_credential
+```
```

Comment:
Just a suggestion, feel free to ignore. Might be nice to add the alternative command with an encoded credential as well.

Reply:
Added to the "Other Examples" tabs below.